src/components/study-pane.js

```diff
@@ -689,9 +711,22 @@ const StudyPane = (props) => {
     useEffect(() => {
         websocketExpectedCloseRef.current = false;
         dispatch(openStudy(studyUuid));
-        loadNetwork();
-        loadGeoData();
-        loadTree();
+
+        fetchStudyExists(studyUuid).then((response) => {
+            if (response.status === 400 || response.status === 404) {
+                setStudyNotFound(true);
+                //since the study is not found no need to try to load network and geoData
+                setStudyExistsChecked(true);
+                setIsNetworkLoadingDone(true);
```

Comment:
Strange to put this. Shouldn't be : 
`setIsNetworkLoadingDone(false);` ?

Reply:
nope because the point is we don't want to do the loading of a network we know doesn't exists